Sources/StreamChat/Workers/CurrentUserUpdater.swift

```diff
@@ -161,6 +161,33 @@ class CurrentUserUpdater: Worker {
             completion?($0.error)
         }
     }
+    
+    /// Get all blocked users.
+    ///
+    /// - Parameter completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    ///
+    func getBlockedUsers(completion: @escaping (Result<[BlockedUser], Error>) -> Void) {
+        apiClient.request(endpoint: .loadBlockedUsers()) {
+            switch $0 {
+            case let .success(payload):
+                var blockedUsers: Set<BlockedUserDTO> = []
+                self.database.write({ session in
+                    for blockedUser in payload.blockedUsers {
+                        blockedUsers.insert(blockedUser.asDTO(context: self.database.writableContext))
+                    }
+                    session.currentUser?.blockedUsers = blockedUsers
+                    session.currentUser?.user.blockedUserIds = blockedUsers.map { $0.blockedUserId }
```

Comment:
You need to map the DTOs here to the models and set the property
```suggestion
                    session.currentUser?.user.blockedUserIds = blockedUsers.map { $0.blockedUserId }
                    blockedUsers = blockedUsers.compactMap { try? $0.asModel() }
```